src/sentry/static/sentry/app/views/settings/organizationTeams/teamMembers.tsx

```diff
@@ -189,19 +198,20 @@ class TeamMembers extends React.Component {
    *
    * @param {Event} e React Event when member filter input changes
    */
-  handleMemberFilterChange = e => {
+  handleMemberFilterChange = (e: React.ChangeEvent<HTMLInputElement>) => {
     this.setState({dropdownBusy: true});
     this.debouncedFetchMembersRequest(e.target.value);
   };
 
-  renderDropdown = access => {
+  renderDropdown = () => {
     const {organization, params} = this.props;
+    const {access} = organization;
     const existingMembers = new Set(this.state.teamMemberList.map(member => member.id));
 
     // members can add other members to a team if the `Open Membership` setting is enabled
     // otherwise, `org:write` or `team:admin` permissions are required
     const hasOpenMembership = organization && organization.openMembership;
-    const hasWriteAccess = access.has('org:write') || access.has('team:admin');
+    const hasWriteAccess = access.includes('org:write') || access.includes('team:admin');
```

Comment:
maybe you can make it a parameter since we are already checking this before calling the `renderDropdown ` function?